app/services/whatsapp/providers/whatsapp_cloud_service.rb

```diff
@@ -1,9 +1,13 @@
 class Whatsapp::Providers::WhatsappCloudService < Whatsapp::Providers::BaseService
+  require 'json'
+
   def send_message(phone_number, message)
     if message.attachments.present?
       send_attachment_message(phone_number, message)
-    else
+    elsif message.content_type == "text"
       send_text_message(phone_number, message)
+    else
+      send_text_message_interactive(phone_number, message)
```

Comment:
since we don't support all interactive messages lets have the case-specific for the one we support 
```suggestion
    elsif message.content_type == "input_select"
      send_text_message_interactive(phone_number, message)
     else
      send_text_message(phone_number, message)

      
```

Reply:
Good sugestion